src/ppx/ppx_lwt.ml

```diff
@@ -334,16 +334,23 @@ let mapper =
             "Lwt's finally should be used only with the syntax: \"(<expr>)[%%finally ...]\"."
         ))
 
-      | [%expr [%e? lhs] >> [%e? rhs]] ->
+      | [%expr [%e? lhs] >> [%e? rhs]] as e ->
```

Comment:
To get the location of the `>>`, I believe you can either:

1. Change this pattern to manually match on `Pexp_apply` of `Pexp_ident ">>"` (I'm skipping the wrapping location and attributes and structures around each `Pexp_*`). You should then be able to get the location from the `Pexp_ident`.
2. Leave this pattern as is, because it's pretty legible. In the case body below, match again on `Pexp_apply` (what `e` is) and extract the location of the function (it should be a `Pexp_ident ">>"` wrapped in location information).

I don't recall immediately how binary operator application is represented, it's likely that there are actually two `Pexp_apply`s, one nested in the other.